iib/workers/tasks/build.py

```diff
@@ -1028,6 +1043,7 @@ def handle_rm_request(
     overwrite_from_index_token=None,
     distribution_scope=None,
     binary_image_config=None,
+    build_tags=[],
```

Comment:
Should it here None instead?

Reply:
it doesn't matter if it's [] or None. But yeah, I should have been consistent